q.yaml

```
@@ -0,0 +1,36 @@
+package:
+  name: q
+  version: 0.11.4
+  epoch: 0
+  description: "A tiny command line DNS client with support for UDP, TCP, DoT, DoH, DoQ and ODoH."
+  copyright:
+    - license: GPL-3.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - build-base
+      - go~1.20
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/natesales/q
+      tag: v${{package.version}}
+      expected-commit: 83c169f40f0895ae4b5b210f20b0a46fda2e0d9d
+
+  - name: Configure and build
+    runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      # build package as requested by the upstream author
+      go build -ldflags="-s -w -X main.version=$(git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)//;s/-/./g') -X main.commit=$(git rev-parse --verify HEAD) -X main.date=$(date -Idate)"
```

Small suggestion for the version string

```suggestion
      go build -ldflags="-s -w -X main.version=v${{package.version}} -X main.commit=$(git rev-parse --verify HEAD) -X main.date=$(date -Idate)"
```